enterprise/backend/src/metabase_enterprise/audit/pages/queries.clj

```diff
@@ -95,9 +95,9 @@
                [:user_name       {:display_name "Created By",      :base_type :type/Text,    :remapped_from :user_id}]
                [:public_link     {:display_name "Public Link",     :base_type :type/URL}]
                [:cache_ttl       {:display_name "Cache Duration",  :base_type :type/Number}]
-               [:avg_exec_time   {:display_name "Average Runtime", :base_type :type/Integer}]
-               [:total_runtime   {:display_name "Total Runtime",   :base_type :type/Number}]
-               [:query_runs      {:display_name "Query Runs",      :base_type :type/Integer}]
+               [:avg_exec_time   {:display_name "Average Runtime (ms)", :base_type :type/Integer}]
+               [:total_runtime   {:display_name "Total Runtime (ms)",   :base_type :type/Number}]
+               [:query_runs      {:display_name "Query Runs (ms)",      :base_type :type/Integer}]
```

Comment:
Now you can tell people you write in LISP :D

Reply:
Haha it's time to add Clojure to my skills on LinkedIn 😄